### PR TITLE
Adding plusplus

### PR DESCRIPTION
**Pull request by newhavengill**

### CODE REVIEW

**Comment by jnimety** [comment]
@newhavengill what does this do?

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "hubot-scripts": "^2.16.1",
     "hubot-shipit": "^0.2.0",
     "hubot-slack": "^3.3.0",
-    "hubot-table-flip": "^0.1.0"
+    "hubot-table-flip": "^0.1.0",
+	"hubot-plusplus": ">=1.1.5"
```
Comment (jnimety) [inline package.json]: fix your whitespace! :smiley:

**Comment by sukima** [comment]
[plusplus](https://www.npmjs.com/package/hubot-plusplus):

> A hubot script for adding or removing points
> 
> Give (or take away) points from people and things, all from the comfort of your personal Hubot.

**Comment by newhavengill** [comment]
Updated the whitespace.  PlusPlus is a nice hubot script that tracks when someone gives you the equivalent of a thumbs up.  We use it a lot.  `justin++` on finding plusplus.

**Comment by sukima** [comment]
@newhavengill: Off Topic but it appears you have two email accounts:
- `"Justin Gill" <justin.gill@espn.com>`
- `"Justin Gill" (@newhavengil) <jgillcentral@gmail.com>`

So your Pull Request looks like it is from a different user then your Git commits. Might be worth looking into your `~/.gitconfig` depending on which project your working on.

I prefer to have one GitHub identity that I use for both personal and work. It helps be double my GitHub stats because is counts all the work I do in private organization and public projects (I'm a stats snob that way).

Another option is to do personal work as a different user on the computer, via SSH to a server at home, or in a virtualbox for non-work projects so you don't have to keep changing your `~/.gitconfig` all the time.

However, the **best solution** is to [make the username/email local to the git repo](http://www.thebuzzmedia.com/git-tip-git-config-user-name-and-user-email-for-local-not-global-config/).

**Comment by jnimety** [comment]
I'm happy with it. :sheep: it!

**Pull request merged**